app/views/citizens/savings_and_investments/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :navigation do %>
+   <%= link_to 'Back', @back_link, class: 'govuk-back-link', id: 'back' %>
+<% end %>
+
+<% content_for(:page_title) { t('.h1-heading') } %>
+
+<h1 class="govuk-heading-xl"><%= content_for(:page_title) %></h1>
+
+<%= render partial: 'accounts_summary', locals: { bank_accounts: @bank_accounts } if @bank_accounts.any? %>
+
+<%= form_with(model: @form,
+              scope: :savings_amount,
+              url: citizens_savings_and_investment_path,
+              method: :patch,
+              local: true) do |form| %>
+
+              <%= render partial: 'shared/forms/error_summary', locals: { model: @form } %>
```

Comment:
This line should be indents to match the beginning of the previous declaration (so two spaced in from the `form_with` line), rather than inline with the parameters passed into `form_with`